object-filter/src/main/java/org/infinispan/objectfilter/impl/syntax/JPAString.java

```diff
@@ -0,0 +1,10 @@
+package org.infinispan.objectfilter.impl.syntax;
+
+/**
+ * @author anistor@redhat.com
+ * @since 9.0
+ */
+interface JPAString {
+
+   String toJpaString();
```

Comment:
toString is just for debugging. It prints internals of the object in raw form rather than rendering them in jpa. But in many cases these are nearly identical.